test/Test/Node/Buffer/ST.purs

```diff
@@ -0,0 +1,26 @@
+module Test.Node.Buffer.ST (test) where
+
+import Prelude
+
+import Control.Monad.ST (run) as ST
+import Effect (Effect)
+import Effect.Console (log)
+import Node.Buffer.Class (create)
+import Node.Buffer.Immutable as Immutable
+import Node.Buffer.ST (STBuffer, run)
+import Test.Assert (assertEqual)
+import Test.Node.Buffer.Class (testMutableBuffer)
+import Type.Proxy (Proxy(..))
+import Unsafe.Coerce (unsafeCoerce)
+
+test :: Effect Unit
+test = do
+  log "Testing Node.Buffer.ST ..."
+  testMutableBuffer (Proxy :: Proxy (STBuffer _)) (unsafeCoerce ST.run >>> pure)
```

Comment:
I think if we want a function whose type is `forall h a. ST h a -> Effect a`, a better choice is just `unsafeCoerce`, since they do have the same runtime representation.

Reply:
I appreciate that this is a bit of a nitpick though. Feel free to leave this as-is if you prefer.

Reply:
Done in 7a48e3333792fedf85293fbdbfe41a17fb1d1566